plugins/grpc/cpp/v1.50.0/Dockerfile

```diff
@@ -15,9 +15,9 @@ WORKDIR /build
 RUN git clone --depth 1 --branch v1.50.0 --recursive https://github.com/grpc/grpc
 WORKDIR /build/grpc
 RUN bazel build //src/compiler:grpc_plugin_support
-RUN bazel build //src/compiler:grpc_cpp_plugin
+RUN bazel build //src/compiler:grpc_cpp_plugin.stripped
```

Comment:
What is `.stripped` here? Is this a stripped version of the binary with debug symbols, etc. if so, do we need to pass any options to actually get this, or it was always available?

Reply:
See https://github.com/bufbuild/plugins/pull/228#discussion_r1029807101